packages/compass-crud/src/components/bulk-delete-modal.tsx

```diff
@@ -16,6 +16,7 @@ import type { BSONObject } from '../stores/crud-store';
 import { toJSString } from 'mongodb-query-parser';
 import { ReadonlyFilter } from './readonly-filter';
 import ReadonlyDocument from './readonly-document';
+import type { Document } from 'bson';
```

Comment:
Pretty sure bulk delete modal is not expecting `window.document` in props, but a bson `Document` type